components/esm/npcstats.cpp

```diff
@@ -95,6 +95,8 @@ void ESM::NpcStats::load (ESMReader &esm)
 
     esm.getHNT (mSkillIncrease, "INCR");
 
```

Comment:
mSpecIncreases needs to be initialized here. Initializing in blank() is not enough (blank() is only used for ESS-importing at the moment).